src/components/kycStatusScreens/KycDenied.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+import {
+  StatusContentContainer,
+  StatusTitle,
+  StatusIconContainer,
+  StatusText,
+} from '../../styles'
+import { ProviderIds } from '../../types'
+import { providerIdToProviderName } from '../../constants'
+import ErrorIcon from '../../icons/Error'
+
+interface Props {
+  providerId: ProviderIds
+}
+
+export function KycDenied({ providerId }: Props) {
+  const providerName = providerIdToProviderName[providerId]
+  return (
+    <StatusContentContainer>
+      <StatusTitle>Your information has been denied</StatusTitle>
+      <StatusIconContainer>
+        <ErrorIcon />
+      </StatusIconContainer>
+      <StatusText>
+        Your identification information has been denied by {providerName}.
+      </StatusText>
+      <StatusText>
+        If you think this was a mistake, please contact your wallet provider.
```

Comment:
hm maybe contact the provider support email... not sure wallets will want to deal with this

Reply:
we could add provider support emails to provider configs similar to the privacy policy urls